spec/index.bs

```diff
@@ -15647,6 +15648,16 @@ The [=texel block memory cost=] of each of these formats is the same as its
         <td>&checkmark;
         <td colspan=2>16
     <tr><th colspan=9>mixed component width, 32 bits per texel (4-byte [=render target component alignment=])
+    <tr>
+        <td>{{GPUTextureFormat/rgb10a2uint}}
+        <td>{{GPUTextureSampleType/"uint"}}
+        <td>&checkmark;
+        <td>
+        <td>&checkmark;
+        <td>
+        <td>
+        <td>4
+        <td>8
```

Comment:
Do you have a reference for this 8? It seems likely but I don't actually know. I don't think we have a way to look this up except to test the Metal validation layers and see what they do (or have @litherum figure it out)